.gitignore

```diff
@@ -16,3 +16,7 @@ relayer/relayer-*-x64
 addresses.json
 
 node_modules
+.DS_Store
+contracts/contracts/.cache
+.build
+scripts/.DS_Store
```

Comment:
The `.DS_Store` entries should really be moved to your own personal `~/.gitignore` or something.

Reply:
yeah, it is coming all the time, i should check permanent solution for this